<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPIS</class>
 <widget class="QDialog" name="DPIS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lSmall">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left Small:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lRing">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left Ring:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lMiddle">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left Middle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIndex">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left Index:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lThumb">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left Thumb:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rSmall">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right Small:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rIndex">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right Index:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rThumb">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right Thumb:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rRing">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right Ring:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rMiddle">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right Middle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSmallResult">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lRingResult">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lMiddleResult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lIndexResult">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lThumbResult">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rSmallResult">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rRingResult">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rMiddleResult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rIndexResult">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rThumbResult">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userImage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>240</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gifImage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>240</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="storeButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Store</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../new.qrc"/>
 </resources>
 <connections/>
</ui>
